client/src/parent/AddRewardDialog.tsx

```diff
@@ -47,126 +46,87 @@ export function AddRewardDialog(props: Prop) {
     <Dialog open={props.open}>
       <DialogContent>
         <DialogContentText>
-          <Grid
-            container
-            direction="column"
-            justify="center"
-            alignItems="center"
-            spacing={1}
-          >
-            <Grid item>
-              <Formik
-                // init for the complete Formik-Component --> (GET-method in edit szenario)
+          <Formik
+            // init for the complete Formik-Component --> (GET-method in edit szenario)
 
-                initialValues={{
-                  rewardName: "Disneyland (1d)",
-                  rewardValue: 500,
-                  childSelect: true,
-                  childSelect2: false,
-                }}
-                // validating for the complete Formik-Component
-                validate={(RewardValidation) => {
-                  const errors: Partial<RewardValidation> = {};
-                  if (!RewardValidation.rewardName) {
-                    errors.rewardName = "Bitte definieren";
-                  } else if (!RewardValidation.rewardValue) {
-                    errors.rewardValue = "Bitte definieren";
-                  } else if (RewardValidation.rewardValue < 1) {
-                    errors.rewardValue = "Einfach gratis?";
-                  }
-                  return errors;
-                }}
-                // submit-function for the complete Formik-Component
-                onSubmit={(RewardValidation, { setSubmitting }) => {
-                  setSubmitting(false);
-                  alert(JSON.stringify(RewardValidation, null, 2));
-                  // programming below the POST-method for the backend
-                  // (in edit szenario --> PUT-method)
+            initialValues={{
+              name: "",
+              value: 100,
+            }}
+            // validating for the complete Formik-Component
+            validate={(RewardValidation) => {
```

Comment:
Vielleicht könnten wir für die Validierung die [Yup Library](https://github.com/jquense/yup) verwenden?
Aber das kann man auch in einem späteren Schritt noch tun, falls man das will.